test/unit/org/apache/cassandra/locator/InetAddressAndPortSerializerTest.java

```diff
@@ -37,8 +37,10 @@ public void testRoundtrip() throws Exception
         InetAddressAndPort ipv4 = InetAddressAndPort.getByName("127.0.0.1:42");
         InetAddressAndPort ipv6 = InetAddressAndPort.getByName("[2001:db8:0:0:0:ff00:42:8329]:42");
 
-        testAddress(ipv4, MessagingService.VERSION_30);
-        testAddress(ipv6, MessagingService.VERSION_30);
+        testAddress(ipv4, MessagingService.VERSION_40);
+        testAddress(ipv6, MessagingService.VERSION_40);
+        testAddress(ipv4, MessagingService.minimum_version);
+        testAddress(ipv6, MessagingService.minimum_version);
```

Comment:
This will test `VERSION_40` three times, since it's the value of `MessagingService.minimum_version` and `MessagingService.current_version`. However, `VERSION_50` won't be tested. I think we could modify the test to simply use `VERSION_40` and `VERSION_50`, and maybe make sure that `minimum_version` and `current_version` are among them.

Reply:
i can see value in testing both explicit values and the minimum and current defined.

cost is irrelevant here.

so… i'd rather add VERSION_50 and not remove any. wdyt?

Reply:
actually… i should loop through supportedVersions()

Reply:
Yes, `supportedVersions()` is perfect for this.